docs/commands/README.md

```diff
@@ -2,10 +2,23 @@
 
 KAM binaries can be downloaded [https://github.com/redhat-developer/kam/releases/latest](https://github.com/redhat-developer/kam/releases/latest)
```

Comment:
Should we get rid of the word "Pipelines" in the heading?

Reply:
I think so.   There are still quite a few occurrence of "pipelines".

Reply:
Probably not in this PR.